src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -158,11 +270,15 @@ class pt10(Macro):
 
     def run(self, *args, **kwargs):
         pass
-
+  
 
 class pt11(Macro):
     """Macro with list of numbers followed by a motor parameter. The repeat
-    parameter may be defined as first one."""
+    parameter may be defined as first one.
+    Usages, ex.:
+    pt11 ct1 [[1][3]] mot1
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.